lib/tasks/foodsoft_setup.rake

```diff
@@ -34,10 +34,20 @@ namespace :foodsoft do
     setup_development
     setup_database
     start_mailcatcher
-    puts yellow "All done! Your foodcoft should be running smoothly."
+    puts yellow "All done! Your foodsoft should be running smoothly."
```

Comment:
What about "Your Foodsoft installation should be running smoothly." ? (or s/installation/setup/)